src/backend/utils/html/replace-entities.js

```diff
@@ -0,0 +1,21 @@
+const entities = require('entities');
+
+function decode(codeElement) {
+  // decode twice for double encoded entities
+  const result = entities.decodeHTML(codeElement.innerHTML);
+  return entities.decodeHTML(result);
+}
+
+module.exports = function (dom) {
+  if (!(dom && dom.window && dom.window.document)) {
+    return null;
+  }
+
+  let fixedCodeElement = {};
+
+  dom.window.document.querySelectorAll('code').forEach((code) => {
+    fixedCodeElement = decode(code);
+    code.innerHTML = fixedCodeElement;
+  });
+  return String(fixedCodeElement); // return result to the tests
```

Comment:
Should this be `return fixedCodeElement.outerHTML;`?

Reply:
Or am I wrong?  I'm unclear on the `String()` conversion though.

Reply:
I believe that the character entities `>` `<` `&` inside the code elements will return encoded if we do `return fixedCodeElement.outerHTML;` 

The `String()` conversion was just me being explicit. We can *probably* get rid of it and it would be fine.

Reply:
Do you want to add a test that confirms that `typeof result === 'string'` and we'll know for sure?  I bet we don't need it

Reply:
Great idea, I'll add those.

I'm still looking for a better way to return the decoded code elements to the tests without encoding it again.